src/types/sequencing.ts

```diff
@@ -38,6 +44,17 @@ export type Parcel = {
   updated_at: string;
 };
 
+export type ParcelBundle = {
+  channel_dictionary_id: number | null;
+  command_dictionary: CommandDictionary | undefined;
```

Comment:
Is there a reason we're tied directly to a `command_dictionary` here but `id`s for the other 2 dictionary types?

Reply:
opps let me change that to id